### ISSUE TITLE
SteamVR 2.2.0 Bindings Missing in Release Builds

### ISSUE THREAD

**Issue by john-rubin** [ValveSoftware/steamvr_unity_plugin#416]
Greetings!

Upon building my application as a release build, my SteamVR bindings do not seem to register on application start.  If the build is flagged as development build, this does not occur, and all bindings register correctly.

I have confirmed that in the release build, all binding files are exported.  I have found other people reporting similar issues, and the general advice seems to be to replace the default bindings for the application, which I've done many times.  I'm not sure where I've gone wrong. 

TLDR: In the Unity Editor and Development Builds, all custom bindings register appropriately.  In non development builds, my custom bindings are gone.

Any insight would be wonderful!

Thank you!

**Comment by zite** [comment]
Are there any errors when you build?

The binding files are just JSONs, can you open them and see if they look like the right bindings?

**Comment by john-rubin** [comment]
Hey dude.

Turns out, I'm an idiot.  There's no issue.  Sorry to have wasted your time.

The action we were noticing was missing was compiled out of release builds with a DEBUG preprocessor directive.

Major facepalm.

**Comment by zite** [comment]
No problem, glad it got resolved :)
